lib/themes/dosomething/paraneue_dosomething/templates/system/html.tpl.php

```diff
@@ -22,17 +22,17 @@
   <?php print $styles; ?>
 
   <!--[if lte IE 8]>
-      <script type="text/javascript" src="<?php print url(VENDOR_ASSET_PATH . '/html5shiv/dist/html5shiv.min.js'); ?>"></script>
-      <script type="text/javascript" src="<?php print url(VENDOR_ASSET_PATH . '/es5-shim/es5-shim.min.js'); ?>"></script>
-      <script type="text/javascript" src="<?php print url(VENDOR_ASSET_PATH . '/es5-shim/es5-sham.min.js'); ?>"></script>
-      <script type="text/javascript" src="<?php print url(VENDOR_ASSET_PATH . '/respond.js/dest/respond.min.js'); ?>"></script>
+      <script type="text/javascript" src="<?php print '/' . VENDOR_ASSET_PATH . '/html5shiv/dist/html5shiv.min.js' ?>"></script>
+      <script type="text/javascript" src="<?php print '/' . VENDOR_ASSET_PATH . '/es5-shim/es5-shim.min.js' ?>"></script>
+      <script type="text/javascript" src="<?php print '/' . VENDOR_ASSET_PATH . '/es5-shim/es5-sham.min.js' ?>"></script>
+      <script type="text/javascript" src="<?php print '/' . VENDOR_ASSET_PATH . '/respond.js/dest/respond.min.js' ?>"></script>
   <![endif]-->
```

Comment:
Just curious, for the JS files, is there a reason we're not using drupal_add_js()?